components/playground/playground.go

```diff
@@ -603,15 +608,28 @@ func (p *Playground) addInstance(componentID string, cfg instance.Config) (ins i
 	return
 }
 
-func (p *Playground) bootCluster(options *bootOptions) error {
+func (p *Playground) bootCluster(env *environment.Environment, options *bootOptions) error {
 	p.bootOptions = options
 
 	if options.pd.Num < 1 || options.tidb.Num < 1 || options.tikv.Num < 1 {
 		return fmt.Errorf("all components count must be great than 0 (tidb=%v, tikv=%v, pd=%v)",
 			options.pd.Num, options.tikv.Num, options.pd.Num)
 	}
 
-	if options.version != "nightly" && options.version != "" {
+	if options.version == "" {
+		version, _, err := env.V1Repository().LatestStableVersion("tidb")
+		if err != nil {
+			return err
+		}
+		options.version = version.String()
+
+		fmt.Println(color.YellowString(`Warning: no version specified for cluster. Will use the last stable one: %s
```

Comment:
s/Warning: no version specified for cluster. Will use the last stable one/Use latest stable version/

I think this is not a warning, just a info to tell the user which version is run, we can use briefer description.

Reply:
Nice catch.